Legacy/bonej/src/main/java/org/bonej/plugins/ConnectedComponents.java

```diff
@@ -648,11 +646,8 @@ private static void applyLUT(final int[][] particleLabels, final int[][] lut, fi
 	 */
 	private static void addNeighboursToMap(final List<IntHashSet> map, final int[] nbh, final int centre,
 			final int IDoffset) {
-		final int l = nbh.length;
 		int lastNonZero = -1;
-		for (int i = 0; i < l; i++) {
-			final int val = nbh[i];
-
+		for (final int val : nbh) {
```

Comment:
This is a really hot bit of code and minor changes can have performance implications. The longhand standard for loop checking against a `final int` is the fastest usually. Please only change if it's been benchmarked to make a speed improvement.